### PR TITLE
cpu/stm32f4: Fix PWM implementation

### PR DESCRIPTION
**Pull request by fnack**
This PR adapts the current low-level periph PWM implementation to allow timers with less than 4 channels. It also configures a PWM timer for the MSB-IoT that only has one available channel.

### CODE REVIEW

File: boards/msbiot/include/periph_conf.h [hunk 1]
```diff
@@ -70,6 +70,26 @@ extern "C" {
 #define TIMER_1_IRQ_CHAN    TIM5_IRQn
 /** @} */
 
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define PWM_NUMOF           (1U)
+#define PWM_0_EN            1
+#define PWM_MAX_CHANNELS    4
```
Comment (haukepetersen) [inline boards/msbiot/include/periph_conf.h]: For this board you can set this value to '1'

**Comment by haukepetersen** [comment]
looks good to me.

**Comment by fnack** [comment]
I addressed your comment and squashed right away (because it's such a minor change). Is this an ACK if Travis is happy again?

**Comment by haukepetersen** [comment]
yes, ACK (but wait for Travis...)

**Comment by fnack** [comment]
As usual, Travis didn't make it through the finish line in time (last test wasn't completed)... I'll merge anyway because all the other tests were successful and the vtimer_msg_diff test is not affected by these changes!

**Pull request merged**